docs/hlo_op_stats.md

```diff
@@ -6,20 +6,42 @@ valuable for identifying potential targets for performance optimization by
 highlighting the most time-consuming operations within your HLO graph during the
 profiled period.
 
-### Supported Platforms
+![Tensorboard HLO Op Stats](images/hlo_op_stats.png)
 
-TPU: Supported
+### Supported Platforms
 
-GPU: Supported
+Both TPU and GPU are supported.
 
 For GPUs, HLO ops have an N:M relationship with the kernels that actually get
-executed. For statistics at the kernel level, see the GPU Kernel Stats tool.
+executed. For statistics at the kernel level, see the
+[GPU Kernel Stats](gpu_kernel_stats.md) tool.
 
 ### HLO Op Stats Tool Components
 
 The HLO Op Stats tool has the following key components:
 
-*   HLO Operation Statistics Table: This is the primary component, presenting a
+*   **Summarized Charts Section**: This section has charts that summarize the
+    detailed results in the per-op table, in the following ways:
+    *   A pie chart titled **Time per HLO category** that shows the fraction of
+        time consumed by different categories of HLO ops.
+    *   A pie chart titled **Time per HLO operation** that shows the fraction of
+        time consumed by different individual HLO ops (truncated to the top-N
+        ops, with the remaining classified as “Other” to improve readability).
+    *   A pie chart titled **Time spent on rematerialization** that shows the
+        fraction of total time spent on ops that are part of rematerialization;
+        XProf receives this information from compiler metadata associated with
+        the profile.
+    *   A pie chart titled **Time spent on rematerialization per HLO category**
+        that shows HLO op categories where the rematerialization time (if any)
+        goes.
+    *   A chart titled **Time spent on outside compilation**. Outside
+        compilation is a TensorFlow feature that enables certain ops within an
+        XLA computation to transparently run on the host CPU rather than the
+        accelerator device (e.g., `tf.summary` or `tf.print` that requires I/O
+        access that the device does not possess).
```

Comment:
This chart shows up empty for me. Not sure if this is the correct wording or if it should be reviewed.